Error compiling shader

[..]
error: Error compiling shader: shaders/opengl/planetrings.vert:
0(120) : error C7011: implicit cast from "vec4" to "vec2"
OpenGL vendor: NVIDIA Corporation
OpenGL renderer string: GeForce GTX 650/PCIe/SSE2

--- a/data/shaders/opengl/planetrings.vert
+++ b/data/shaders/opengl/planetrings.vert
@@ -8,6 +8,6 @@ void main(void)
 {
 	gl_Position = logarithmicTransform();
 
-	texCoord0 = a_uv0;
+	texCoord0 = a_uv0.xy;
 	texCoord1 = a_vertex;
 }

